prospector/config/configuration.py

```diff
@@ -36,29 +40,29 @@ class Config(BaseModel):
     messages_only: bool = False
     summary_only: bool = False
     quiet: bool = False
-    output_format: Optional[List[Formatters]] = None
+    output_format: Optional[list[Formatters]] = None
     absolute_paths: bool = False
-    tools: Optional[List[Tools]] = None
-    with_tools: List[str] = []
-    without_tools: List[str] = []
-    profiles: List[str] = []
-    profile_path: List[str] = []
+    tools: Optional[list[Tools]] = None
+    with_tools: list[str] = []
+    without_tools: list[str] = []
+    profiles: list[str] = []
+    profile_path: list[str] = []
```

Comment:
This can be done automatically with ruff, we should add it to the pre-commit conf